### DISCUSSION TITLE
Monitoring tool for SQL Server as Transport Layer?

### DISCUSSION THREAD

**Discussion by levinor** [rebus-org/Rebus#1074] [Q&A] [answered]
Hi all!

I have an application using Rebus with SQL Server as a transport layer (yes, I know this is not the ideal transport layer) and I'm, trying to find an out-of-the-box tool to do real-time monitoring of the queues. I tried Rebus Snoop but I found that it is not compatible with SQL Server.

Is there any tool that I can use?

Thank you very much.

**Comment by mookid8000** [comment, marked as answer]
Unfortunately, Rebus doesn't have anything "out-of-the-box" to help with that.

[Rebus Pro](https://pro.rebus.fm/) customers can use Fleet Manager to see what's going on. It works with all transports and can even help with retrying failed messages, call webhooks or post to Slack when stuff fails, etc. You can check out the features here: https://pro.rebus.fm/#fleet-manager-features

If you're after something free, I suggest you prepare a couple of SQL scripts to e.g. retry failed messages, etc., so you have them ready for when you need them.
